mvrx/src/test/kotlin/com/airbnb/mvrx/BaseTest.kt

```diff
@@ -49,26 +38,42 @@ abstract class BaseTest {
         }
     }
 
+    @Before
+    @After
+    fun resetConfigurationDefaults() {
+        MvRx.viewModelConfigFactory = MavericksViewModelConfigFactory(true, contextOverride = TestCoroutineDispatcher())
```

Comment:
Should this be separated into `before` and `after` functions and after set it to null? I think a general principle should be to return static variables to their default state in tests.

Reply:
+1